src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/UpstreamApi.kt

```diff
@@ -1,5 +1,5 @@
 package uk.gov.justice.digital.hmpps.hmppsintegrationapi.models
 
 enum class UpstreamApi {
-  NOMIS, PRISONER_OFFENDER_SEARCH, PROBATION_OFFENDER_SEARCH, NDELIUS
+  NOMIS, PRISONER_OFFENDER_SEARCH, PROBATION_OFFENDER_SEARCH, NDELIUS, ARN
```

Comment:
```suggestion
  NOMIS, PRISONER_OFFENDER_SEARCH, PROBATION_OFFENDER_SEARCH, NDELIUS, ASSESS_RISKS_AND_NEEDS
```

Could we use the full name of the API like we've done with the other APIs to remove the cognitive load (and me thinking Amazon Resource Name)? 😝

Reply:
Yeah good thinking, will change!